cmd/commands/destroy_cluster.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+	"github.com/rook/kubectl-rook-ceph/pkg/crds"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+// DestroyClusterCmd represents the command for destroy cluster
+var DestroyClusterCmd = &cobra.Command{
+	Use:   "destroy-cluster",
+	Short: "delete all Rook CRDs",
+	Args:  cobra.NoArgs,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		clientsets := GetClientsets(cmd.Context())
+		destroyClusterConfirmation, _ := cmd.Flags().GetBool("yes-really-destroy-cluster")
+		if !destroyClusterConfirmation {
+			fmt.Println("Error: you need to provide the flag --yes-really-destroy-cluster to confirm the cluster deletion")
+			cmd.Usage()
+			return
+		}
+		logging.Info("removing crds")
+		err := crds.DeleteCustomResources(cmd.Context(), clientsets, CephClusterNamespace)
+		if err != nil {
+			logging.Fatal(err)
+		}
+		logging.Info("done")
+	},
```

Comment:
```suggestion
		crds.DeleteCustomResources(cmd.Context(), clientsets, CephClusterNamespace)
	},
```

can you check other command formats, basically we don't handle error in command package, we directly handle from pkg/  directory only.

Reply:
Hey @subhamkrai I have a question here usually is good to have one single place to treat the errors that occurs inside the package I can change it of course but what do you think about that?.

By other side if I remove the return error, all my unittest will be useless since they cannot to use the return of the function to validate if works properly or not.

Let me know your thoughts